templates/list_pages/node--oe-list-page--full.html.twig

```diff
@@ -25,10 +25,14 @@
     <div class="col-12 col-lg-9 col-xxl-8">
       <div class="row">
         <div class="col-md-6 col-lg-8 align-self-center">
-          <h4 class="mb-0">
-            <span class="text-capitalize">{{ label }}</span>
-            ({{ content['extra_field_oe_list_page_total_count:node:oe_list_page'] }})
-          </h4>
+          {% set _title %}
+            {{ label|render|striptags }}({{ content['extra_field_oe_list_page_total_count:node:oe_list_page'] }})
+          {% endset %}
```

Comment:
We miss a space before the '('.
On the other hand, we get free space from the field template. The `|striptags` removes tags, but not spaces.
The `apply spaceless` that is applied to the entire template only removes space between tags, not within tags, so we should add the `-` here.
Let's do this in a cleaner way so we get exactly one space before `(` but no extra space elsewhere.
```suggestion
          {% set _title -%}
            {{ label|field_value }} ({{ content['extra_field_oe_list_page_total_count:node:oe_list_page'] }})
          {%- endset %}
```
I did a `{{ dump(label['#cache']) }}` to confirm there is no cache data we would lose by calling `|field_value`. And we already plan to fix cache handling for `|field_value` anyway.

Reply:
Following our discussion from last week, we could move this change to a follow-up and leave the template unchanged in this issue.